examples/audio.rs

```diff
@@ -0,0 +1,126 @@
+//! # Example: Audio
+//!
+//! This example allows you to
```

Comment:
There should be some explanation of why this example exists (to show how audio can be integrated and more broadly how other SDL subsystems can be used). If you don't want to write it let me know and I'll have a go at it.

Some additional comments inside the code, e.g. when the audio subsystem is initialized, would also help.